game/models.py

```diff
@@ -179,6 +142,21 @@ def save(self, *args, **kwargs):
             self.created_at = timezone.now()
         return super(Task, self).save(*args, **kwargs)
 
+class DeployedTaskImage(models.Model):
```

Comment:
This model is designed to represent a single host/port pair, but it is possible to have multiple host/port pairs for a single docker image. I propose to rename the model to something like ImageConnection.